servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRest.java

```diff
@@ -17,8 +17,18 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Please move all the non-GC related changes in AbstractRest hierarchy to a separate PR